src/components/EReceiptThumbnail.js

```diff
@@ -37,12 +36,9 @@ const backgroundImages = {
     [CONST.ERECEIPT_COLORS.PINK]: eReceiptBGs.EReceiptBG_Pink,
 };
 
-function getBackgroundImage(transaction) {
-    return backgroundImages[StyleUtils.getEReceiptColorCode(transaction)];
-}
-
 function EReceiptThumbnail({transaction}) {
     const styles = useThemeStyles();
+    const StyleUtils = useThemeStyles();
```

Comment:
This is a typo, right? You're using the same hook twice.

Reply:
Yes, 
```suggestion
    const StyleUtils = useStyleUtils();
```

Reply:
yes 👍